### PR TITLE
Correct Category Page Footer Text Background

### PR DESCRIPTION
**Pull request by AnushkaChouhan25**
### Pull Request Checklist
Issue - #1541 

image

- [x] I have added screenshots and videos to show before and after the working of my code.
- [x] I have ensured that the screen size is set to 100% while making the video.
- [x] I have synced the latest fork with my local repository and resolved any conflicts.
- [x] I have mentioned the issue number which I created before making this PR .(format to mention issue number is : fixes #(issue number) ) 
- [x] I understand that if any the above conditions are not met , my PR will not be MERGED .

### CODE REVIEW

**Comment by ANSHIKA-26** [comment]
sync fork

File: backend/app.js
Comment (ANSHIKA-26) [inline backend/app.js:1]: why have you removed this route ?

**Comment by ANSHIKA-26** [comment]
hey @AnushkaChouhan25 there are a lot of changes done which are not even required , you have removed some routes from backend and also changed some code in backend folder completely irrelevant to your issue ,kindly check

File: backend/models/feedback.js
Comment (ANSHIKA-26) [inline backend/models/feedback.js:1]: why was this feel deleted ?

File: backend/controllers/userController.js
Comment (ANSHIKA-26) [inline backend/controllers/userController.js:1]: lots of code removed from here , why ?

**Review by ANSHIKA-26 — CHANGES_REQUESTED** [review]

**Comment by AnushkaChouhan25** [comment]
Hey @ANSHIKA-26, I have doubts regarding this problem, Can you please help me? 
This happened sometimes I made changes in only one file which was required but all other files also got added in which I did not make any changes, and this happens after committing,
all the files get pulled including the changes which I did not make.
And I use GitHub Desktop to contribute to the repo and face this same problem.

**Comment by ANSHIKA-26** [comment]
try syncing fork , then making a new PR

**Comment by AnushkaChouhan25** [comment]
I always sync pr first then start work, and Not happened with all repos but yes with some other.

**Comment by AnushkaChouhan25** [comment]
Hi @ANSHIKA-26, I made a new pr, Can you review it?

**Pull request closed without merging**